specification/sbi_atee.adoc

```diff
@@ -938,6 +938,202 @@ The possible error codes returned in `sbiret.error` are shown below.
 | SBI_ERR_FAILED          | The operation failed for unknown reasons.
 |===
 
+[#sbi_tee_host_tvm_block_pages]
+=== Function: TEE Host TVM Block Pages (FID #16)
+[source, C]
+-----
+struct sbiret sbi_tee_host_tvm_block_pages(unsigned long tvm_guest_id,
+                                           unsigned long guest_page_address,
+                                           unsigned long length);
+-----
+
+Invalidates the pages in the specified range of guest physical address space and
+marks the page as either `blocked` or `BlockedShared`.
```

Comment:
There was no reference to these states earlier. If we want to use these states, a state diagram as described in the design doc would be good. 

Or generic descriptions should be provided.

Reply:
Commits removed. I will be opening a new PR for that after reworking the share/unshare parts.